Mesh_3/examples/Mesh_3/CMakeLists.txt

```diff
@@ -73,8 +73,11 @@ if ( CGAL_FOUND )
     create_single_source_cgal_program( "mesh_cubes_intersection_with_features.cpp" )
     create_single_source_cgal_program( "mesh_implicit_domains.cpp" "implicit_functions.cpp" )
     create_single_source_cgal_program( "mesh_polyhedral_domain.cpp" )
+    create_single_source_cgal_program( "mesh_polyhedral_domain_sm.cpp" )
     create_single_source_cgal_program( "remesh_polyhedral_surface.cpp" )
     create_single_source_cgal_program( "mesh_polyhedral_domain_with_features.cpp" )
+    create_single_source_cgal_program( "mesh_polyhedral_domain_with_features_sm.cpp" )
+    create_single_source_cgal_program( "mesh_polyhedral_domain_with_two_polyhedra.cpp" )
```

Comment:
@afabri Are those new example just supposed to exist, without any explanation/documentation for users?